### PR TITLE
Use ansible 2.6

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

**Comment by sdodson** [comment]
Bump openshift-ansible.spec Requires?

**Comment by vrutkovs** [comment]
Spec bumped

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Install base dependencies:
 
 Requirements:
 
-- Ansible >= 2.4.3.0, 2.5.x is not currently supported for OCP installations
+- Ansible >= 2.4.3.0, 2.5.x+ are not currently supported for OCP installations
```
Comment (sdodson) [inline README.md]: Don't we want to change this? Perhaps with a note that it's currently only required for master / 3.11?
` - Ansible >= 2.6.0`

Comment (mtnbikenc) [reply README.md]: Agreed.
`- Ansible >= 2.6.0`

**Comment by vrutkovs** [comment]
All done, I'll squash commits later

**Comment by vrutkovs** [comment]
CI repos don't have ansible 2.6 yet. @stevekuznetsov @smarterclayton  could you help us with that?

**Comment by sdodson** [comment]
> CI repos don't have ansible 2.6 yet. @stevekuznetsov @smarterclayton could you help us with that?

there are ansible 2.6 repos in the mirrors, just need to enable them. somewhere in aos-cd-jobs we're probably enabling ansible-2.4 repo, we need to update that to enable 2.6 repo for master/release-3.11

**Comment by DanyC97** [comment]
@vrutkovs @sdodson i suspect this PR won't be backported to 3.10 and as such i guess don't need to worry about having 2.6 tagged into the PaaS 3.10 repo ?

**Comment by vrutkovs** [comment]
>i suspect this PR won't be backported to 3.10

Correct, 3,10 stays on 2.4

**Pull request merged**